nullaway/src/main/java/com/uber/nullaway/Config.java

```diff
@@ -39,7 +39,8 @@
   boolean serializationIsActive();
 
   /**
-   * Getter method for {@link FixSerializationConfig}.
+   * Getter method for {@link FixSerializationConfig}. Fix Serialization feature must be activated,
+   * otherwise calling this method will fail the execution.
```

Comment:
```
* Getter method for {@link FixSerializationConfig}.
*
* <p>Fix Serialization feature must be activated, otherwise calling this method will fail 
* the execution.
```